dev/tools/controllerbuilder/generate-proto.sh

```diff
@@ -68,7 +68,10 @@ protoc --include_imports --include_source_info \
     ${REPO_ROOT}/mockgcp/apis/mockgcp/cloud/apigee/*/*.proto \
     ${REPO_ROOT}/mockgcp/apis/mockgcp/cloud/networkconnectivity/*/*.proto \
     ${THIRD_PARTY}/googleapis/google/*/*.proto \
-    ${THIRD_PARTY}/googleapis/google/api/*.proto \
+    ${THIRD_PARTY}/googleapis/google/*/*/*.proto \
```

Comment:
I believe these 4 new lines make lined 75-89 redundant.
Any concerns with us including versions other than just the ones we want? (Eg. monitoring older than v3)